### ISSUE TITLE
.snap files using scope injections only display results from a single scope

### ISSUE THREAD

**Issue by gnikit** [PanAeon/vscode-tmgrammar-test#29]
## Issue

I have been trying to generate some .snap files for grammars that use scope injections from other scopes (in the `package.json`) but I have been unsuccessful. So for example scope `A` defined in `a.json` is injected to scope `B` defined in `b.json` into `B`'s `  "injectionSelector": "comment"`. 
Then when generating the .snap file only scopes belonging to `B` are present. Am I missing something?

The actual syntax highlighting works as expected in VSCode.

## Expected behaviour

The .snap file should contain scopes from both `A` and `B`

![Screenshot from 2021-10-03 23-25-29](https://user-images.githubusercontent.com/16143716/135773519-d7cc714a-5582-4f5d-97ba-9b74a972a3f7.png)

## Setup

A more detailed setup to understand the problem

### **package.json**

Consider the following `"grammars"` setup in the `package.json` of an extension, where we inject `openacc_lang.json` with scope `source.openmp` into `source.fortran.free` the main scope of this language

```json
    "grammars": [
      {
        "language": "FortranFreeForm",
        "scopeName": "source.fortran.free",
        "path": "./syntaxes/fortran_free-form.tmLanguage.json"
      },
      {
        "scopeName": "source.openacc",
        "path": "./syntaxes/openacc_lang.json",
        "injectTo": ["source.fortran.free"]
      }
    ]
```

### Generating .snap

to generate the .snap file

```
vscode-tmgrammar-snap -u -s source.fortran.free -g fortran_free-form.tmLanguage.json -g openacc_lang.json -g  -t openacc_support.f90
```

## Files

files (.json and test case) needed to generate a .snap file

[bug-openacc-snap.tar.gz](https://github.com/PanAeon/vscode-tmgrammar-test/files/7274612/bug-openacc-snap.tar.gz)

**Comment by gnikit** [comment]
Out of curiosity, is there any progress on this? It would certainly make it a lot easier maintaining the grammar for the Modern Fortran extension

**Comment by PanAeon** [comment]
@gnikit, sorry for not responding for so long, I must have completely forgotten about this issue.  The thing is that injection grammars are not supported yet. I'll look into this tomorrow, looks like`vscode-textmate` library exposes some API for the injections, if so it is just a matter of passing correct configuration to it.

**Comment by gnikit** [comment]
Awesome @PanAeon, thanks a lot!
Fixed in [v0.1.1](https://github.com/PanAeon/vscode-tmgrammar-test/releases/tag/v0.1.1)
